.github/workflows/surge-ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: olafurpg/setup-scala@v10
         with:
-          java-version: openjdk@1.11.0
+          java-version: openjdk@1.15.0
```

Comment:
I tried to go to 16 but unfortunately something fails with 16 and getting weird errors ...